servicecontrol/servicecontrol-instances/remotes.md

```diff
@@ -161,6 +161,10 @@ In this deployment, each region has a full ServiceControl installation with a pr
 
 A new cross-region primary instance is added to allow ServiceInsight to show messages from both regions. This cross-region instance includes each region-specific primary instance as a remote allowing it to query messages from both. The cross-region instance should disable error queue management by configuring the [error queue](/servicecontrol/creating-config-file.md#transport-servicebuserrorqueue) with the value `!disable`.
 
+### Zero downtime upgrades
+
+The remotes feature can be used to perform [zero downtime upgrades](/servicecontrol/upgrades/zero-downtime.md).
```

Comment:
Should we also mention that this can be used to deal with retention as well or is that to advanced?